lib/mongodb-filter-wrapper.js

```diff
@@ -67,6 +67,23 @@ class MongoDBFilterWrapper {
 		return parsedFilters;
 	}
 
+	/**
+		* Validates if exists another field and merge with and condition
+		* @param {Object} filter One filter or object given
+		* @param {Object} modelField Model field that use in filters
+		* @param {Object} filterToResponse Filter that will be returned
+		* @return {string|Object} Filter created or merged object
+		*/
+	static createObjectFilter(filter, modelField, filterToResponse) {
+
+		let filterGenerated = this.createFilter(filter, modelField);
+
+		if(modelField && modelField.field && filterToResponse[modelField.field])
+			filterGenerated = Object.assign(filterGenerated, filterToResponse[modelField.field]);
+
+		return filterGenerated;
```

Comment:
Toda la función podria ser reemplazada asi:

```suggestion
		return ({ ...this.createFilter(filter, modelField), ...filterToResponse[modelField.field] });
```

PD: `createFilter`, el actual método, no confundir con mi comentario anterior